composer/loggers/console_logger.py

```diff
@@ -162,7 +162,8 @@ def eval_start(self, state: State, logger: Logger) -> None:
         # Remove index of last batch, so that we don't print progress at end of last batch and then
         # at eval end.
         last_batch_idx = total_eval_batches
-        self.eval_batch_idxs_to_log.remove(last_batch_idx)
+        if last_batch_idx in self.eval_batch_idxs_to_log:  # eval_batch_idxs_to_log may be empty
```

Comment:
tiny nit, but if the only scenario we're checking here is when its empty, lets just have the if statement check if its empty?

Reply:
I'm going to leave as is since it's a little stronger than known edge cases but I'd rather catch and avoid a scenario we haven't envisioned vs. raise an error